deploy.sh

```
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+EC2_HOST="ubuntu@ec2-3-36-57-197.ap-northeast-2.compute.amazonaws.com"
+PEM_KEY_PATH="/Users/spqje/key-jenson.pem"
+TARGET_DIR="~/home/ubuntu/app"
```

Shellcheck (0.10.0)

[warning] Tilde does not expand in quotes. Use $HOME.

(SC2088)